src/content/developers/docs/nodes-and-clients/run-a-node/index.md

```diff
@@ -19,209 +19,470 @@ If you're new to the topic of running a node, or looking for a less technical pa
+
+#### Requirements {#requirements}
+
+Hardware requirements differ by client but generally are not that high since the node just needs to stay synced. Don't confuse it with mining, which requires much more computing power. Sync time and performance do improve with more powerful hardware however. 
+
+Before installing any client, please ensure your computer has enough resources to run it. Minimum and recommended requirements can be found below. 
+
+The hardware bottleneck is mostly a disk. Syncing the Ethereum blockchain is very input/output intensive and requires a lot of space. It is best to have a solid-state drive (SSD) with hundreds of GBs of free space to spare even after the synchronization. To run an Ethereum client on an HDD, you will need at least 8GB of RAM to use as a cache.
+
+Size of the database and speed of the initial synchronization depends on chosen client, its configuration and [sync strategy](/developers/docs/nodes-and-clients/#sync-modes). 
 
-Check the minimum and recommended [disk space requirements for each client and sync mod](/developers/docs/nodes-and-clients/#requirements).
-Generally, modest computing power should be enough. The problem is usually drive speed. During initial sync, Ethereum clients perform a lot of read/write operations. Therefore SSD is strongly recommended. A client might not even [be able to sync current state on HDD](https://github.com/ethereum/go-ethereum/issues/16796#issuecomment-391649278) and get stuck a few blocks behind Mainnet.
-You can run most of the clients on a [single board computer with ARM](/developers/docs/nodes-and-clients/#ethereum-on-a-single-board-computer/). You can also use the [Ethbian](https://ethbian.org/index.html) operating system for Raspberry Pi 4. This lets you [run a client by flashing the SD card](/developers/tutorials/run-node-raspberry-pi/).
-Based on your software and the hardware choices, the initial synchronization time and storage requirements may vary. Be sure to [check sync times and storage requirements](/developers/docs/nodes-and-clients/#recommended-specifications).
 Also make sure your internet connection is not limited by a [bandwidth cap](https://wikipedia.org/wiki/Data_cap). It's recommended to use an unmetered connection since initial sync and data broadcasted to the network could exceed your limit.
 
-#### Operating system {#operating-system}
+##### Operating system {#operating-system}
 
 All clients support major operating systems - Linux, MacOS, Windows. This means you can run nodes on regular desktop or server machines with the operating system (OS) that suits you the best. Make sure your OS is up to date to avoid potential issues and security vulnerabilities.
 
+##### Minimum requirements {#recommended-specifications}
+
+- CPU with 2+ cores
+- 4 GB RAM or 8GB with HDD 
```

Comment:
Is this true post-merge? Seems low?

Reply:
Yeah it should be at least 8GB RAM. And I would remove the HDD references, I think should not recommend that